### DISCUSSION TITLE
Change font of KaTeX

### DISCUSSION THREAD

**Discussion by Yue-Zhengyuan** [KaTeX/KaTeX#3716] [Q&A] [answered]
Is it possible to change the font of KaTeX from the default [Computer Modern](https://en.wikipedia.org/wiki/Computer_Modern) to Times New Roman (or [STIX](https://www.stixfonts.org))? (I just personally don't quite like the Computer Modern font. In LaTeX this change of font can be done by adding `\usepackage{newtxmath}` to the preamble.)

**Comment by ronkok** [comment, marked as answer]
KaTeX is quite tightly bound to its own KaTeX fonts. It calls characters by the code-points in the KaTeX fonts and it sets positions of math elements using calculations that rely on the glyph widths and heights in the KaTeX fonts. The use of a different font would be a major project.
